src/kernel/mconfig_dot.ml

```diff
@@ -283,6 +290,7 @@ let prepend_config ~dir:cwd configurator (directives : directive list) config =
     | `UNKNOWN_TAG tag  ->
       let error =  Printf.sprintf "Unknown configuration tag \"%s\"" tag in
       config, error :: errors
+    | `TAG_ERROR error -> config, error :: errors
```

Comment:
The other error case prints a message - probably we should too?

Reply:
It's a `sprintf` not a `printf` - it's just constructing a string to print later

Reply:
Oops - sorry for the noise!